.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 exclude: ^(doc/_static/.*|doc/_themes/.*)$
 repos:
   - repo: https://github.com/saltstack/pip-tools-compile-impersonate
-    rev: master
+    rev: 999c70e967636c579e919bc2bb30985b4c9e996d
```

Comment:
This should stay as `master`

Reply:
Yes, it should. Why did you need to change it?